src/addTaskToDisplay.js

```diff
@@ -0,0 +1,124 @@
+      e.target.style.textDecoration = 'line-through';
+    }
+  });
+
+  taskinput.addEventListener('keyup', (e) => {
+    const editedText = e.target.value;
+    const index = e.target.getAttribute('data-index');
+    taskObj.editTask(index, editedText);
+  });
+
+  taskinput.addEventListener('keypress', (e) => {
+    if (e.key === 'Enter') {
+      if (e.target.classList.contains('focused')) {
+        e.target.parentNode.style.backgroundColor = '';
+        e.target.style.backgroundColor = '';
+        e.target.nextElementSibling.setAttribute('src', Dots);
+        e.target.classList.remove('focused');
+        if (e.target.previousElementSibling.checked) {
+          e.target.style.textDecoration = 'line-through';
+        }
+        document.getElementById('input-task').focus();
+      }
+    }
+  });
+};
+
+const addCheckboxListener = (checkbox) => {
+  checkbox.addEventListener('change', (e) => {
+    if (e.target.checked) {
+      const boxIndex = e.target.getAttribute('data-index');
+      e.target.nextElementSibling.style.textDecoration = 'line-through';
+      taskObj.markAsCompleted(boxIndex);
+    } else {
+      // console.log("its not checked");
+      e.target.nextElementSibling.style.textDecoration = 'none';
+    }
+  });
+};
+
+const addRemoveButtonListener = (removebtn) => {
+  removebtn.addEventListener('mousedown', (e) => {
+    e.preventDefault();
+    const dataIndex = e.target.getAttribute('data-index');
+    if (e.target.previousElementSibling.classList.contains('focused')) {
+      taskObj.removeTask(dataIndex);
+      removeTaskFromDisplay(dataIndex);
+      const tasks = document.querySelectorAll('.dots');
+      const chkboxes = document.querySelectorAll('.checkbox');
+      const taskInputs = document.querySelectorAll('.task-desc');
+      const index = dataIndex - 1;
+      for (let i = index; i < taskObj.tasksArr.length; i += 1) {
+        tasks[i].setAttribute('data-index', i + 1);
+        chkboxes[i].setAttribute('data-index', i + 1);
+        taskInputs[i].setAttribute('data-index', i + 1);
+      }
+    }
+  });
+};
+
+const addTaskToDisplay = (task) => {
+  const li = document.createElement('li');
+  const checkBoxInput = document.createElement('input');
+  const descInput = document.createElement('input');
+  checkBoxInput.setAttribute('type', 'checkbox');
+  checkBoxInput.setAttribute('data-index', task.index);
+  descInput.setAttribute('type', 'text');
+  descInput.setAttribute('data-index', task.index);
+
+  // add image to list item
+  const dotImage = new Image();
+  dotImage.src = Dots;
+  dotImage.classList.add('dots');
+  dotImage.setAttribute('data-index', task.index);
+  li.classList.add('task-item');
+  checkBoxInput.classList.add('checkbox');
+  descInput.classList.add('task-desc');
+  descInput.setAttribute('value', task.description);
+
+  li.appendChild(checkBoxInput);
+  li.appendChild(descInput);
+  li.appendChild(dotImage);
+  ul.appendChild(li);
```

Comment:
Accoriding to the DRY rule this could be redeuced to one line of code `ul.appendChild(checkBoxInput,descInput.....)`